data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/ManagementDisabledIndexManagerTest.java

```diff
@@ -117,10 +117,14 @@ void getIndexAlias_IndexWithTimePattern() {
 
     @Test
     void getIndexAlias_IndexWithTimePattern_Exceptional_NotAsSuffix() {
-        when(indexConfiguration.getIndexAlias()).thenReturn(indexAliasWithTimePattern + "randomtext");
-        assertThrows(IllegalArgumentException.class,
-                () -> indexManagerFactory.getIndexManager(
-                        IndexType.MANAGEMENT_DISABLED, openSearchClient, restHighLevelClient, openSearchSinkConfiguration, templateStrategy));
+        when(indexConfiguration.getIndexAlias()).thenReturn(indexAliasWithTimePattern + "-randomtext");
+        final IndexManager objectUnderTest = indexManagerFactory.getIndexManager(
+                IndexType.MANAGEMENT_DISABLED, openSearchClient, restHighLevelClient, openSearchSinkConfiguration, templateStrategy);
+        final Pattern expectedIndexPattern = Pattern.compile(baseIndexAlias + "-\\d{4}.\\d{2}.\\d{2}.\\d{2}-randomtext");
+        try {
+            final String actualIndexPattern = objectUnderTest.getIndexName(null);
+            assertThat(actualIndexPattern, matchesPattern(expectedIndexPattern));
+        } catch (IOException e){}
```

Comment:
Similar comment about this catch statement as above.

Reply:
Copied from another test. Fixed both.